packages/client/public/locales/mn/home.json

```
@@ -0,0 +1,79 @@
+    "chooseOpponent": "Өрсөлдөгчөө сонгоно уу",
+    "apprentice": "Шавь",
+    "apprenticeDesc": "Сул тоглодог, алдаа гаргадаг.",
+    "gentleman": "Жентлмен",
+    "gentlemanDesc": "Ухаалаг тоглодог, заримдаа алдаа гаргана.",
+    "highRoller": "Өндөр тоглогч",
+    "highRollerDesc": "Хайр найргүй. Бүх картыг тооцдог."
+  },
+  "custom": {
+    "title": "Захиалгат ширээ",
+    "create": "Үүсгэх",
+    "join": "Нэгдэх",
```

Biome (2.4.15)

[error] The key join was already declared.

(lint/suspicious/noDuplicateObjectKeys)